app/controllers/application_controller.rb

```diff
@@ -26,7 +26,8 @@ def set_current_user
     @current_user = ArtsyAdminAuth.decode_user(session[:access_token])
   end
 
-  def set_raven_context
-    Raven.user_context(id: @current_user)
+  def set_sentry_context
+    # Heads up! @current_user is the user's id as a string not an object
+    Sentry.set_user(user_id: @current_user)
   end
```

Comment:
Didnt want to mess with `@current_user` at this time - it is used everywhere 
and its technically a STRING and not an object - bit of a gotcha